test/integration/minishift.go

```diff
@@ -56,6 +56,15 @@ func (m *Minishift) shouldHaveState(expected string) error {
 	return nil
 }
 
+func (m *Minishift) shouldHaveProfileState(profile string, expected string) error {
```

Comment:
looks good to me, +1

Reply:
I think it is more "profileShouldHaveState" or maybe even better "ProfileShouldBeInState".

Reply:
Yup, second one is even better however i feel “profileShouldHaveState” is more align with function like ```shouldHaveState```. Also in feature context I find the similar kind of alignment in naming of function. So I think It would be great to keep ```profileShouldHaveState```

Reply:
>  So I think It would be great to keep profileShouldHaveState

I am fine with that, but this is not what you have. You have 'shouldHaveProfileState'.

Reply:
This really reads as "profile state" which could conjure the idea of a new type of state specific to a profile. What we are after is the state of the Minishift VM for a given profile.